src/content/docs/hi/develop-and-build.mdx

```diff
@@ -0,0 +1,145 @@
+
+ब्राउज़र में अपनी साइट देखते समय, आपके पास [एस्ट्रो डेव टूलबार](/hi/guides/dev-toolbar/) तक पहुंच होगी। जैसे-जैसे आप निर्माण करेंगे, यह आपको अपने [इसलैंड्स](/hi/concepts/islands/) का निरीक्षण करने, पहुंच-योग्यता संबंधी समस्याओं का पता लगाने और बहुत कुछ करने में मदद करेगा।
+
+यदि आप डेव सर्वर शुरू करने के बाद ब्राउज़र में अपना प्रोजेक्ट नहीं खोल पा रहे हैं, तो उस टर्मिनल पर वापस जाएँ जहाँ आपने `dev` कमांड चलाया था और प्रदर्शित संदेश की जाँच करें। यह आपको बताएगा कि क्या कोई त्रुटि हुई है, या क्या आपका प्रोजेक्ट [http://localhost:4321/](http://localhost:4321/) के अलावा किसी अन्य URL पर प्रस्तुत किया जा रहा है।
+
+## अपनी साइट बनाएं और उसका पूर्वावलोकन करें
+
+बिल्ड समय पर बनाई जाने वाली आपकी साइट का वर्शन जांचने के लिए, डेव सर्वर से बाहर निकलें (<kbd>Ctrl</kbd> + <kbd>C</kbd>) और अपने टर्मिनल में अपने पैकेज मैनेजर के लिए उपयुक्त बिल्ड कमांड चलाएँ:
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm run build
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm build
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run build
+      ```
+      </TabItem>
+    </Tabs>
+
+एस्ट्रो एक अलग फ़ोल्डर (डिफ़ॉल्ट रूप से `dist/`) में आपकी साइट का एक डिप्लॉय-रेडी वर्शन बनाएगा और आप टर्मिनल में इसकी प्रगति देख सकते हैं। यह आपको उत्पादन में डिप्लॉय करने से पहले आपके प्रोजेक्ट में किसी भी बिल्ड एर्रोर्स के बारे में सचेत करेगा। यदि टाइपस्क्रिप्ट को `strict` या `strictest` पर कॉन्फ़िगर किया गया है, तो `build` स्क्रिप्ट आपके प्रोजेक्ट को टाइप त्रुटियों के लिए भी जाँचेगी।
+
+जब बिल्ड समाप्त हो जाए, तो अपने टर्मिनल में उपयुक्त `प्रीव्यू` कमांड (जैसे `npm run preview`) चलाएं और आप अपनी साइट का निर्मित वर्शन उसी ब्राउज़र प्रीव्यू विंडो में स्थानीय रूप से देख सकते हैं।
+
+ध्यान दें कि यह आपके कोड का प्रीव्यू अर्थात पूर्वावलोकन  करता है जैसा कि बिल्ड कमांड को अंतिम बार चलाने पर मौजूद था। इसका उद्देश्य आपको यह प्रीव्यू देना है कि वेब पर तैनात होने पर आपकी साइट कैसी दिखाई देगी। निर्माण के बाद आपके द्वारा अपने कोड में किए गए कोई भी बाद के परिवर्तन तब तक आपकी साइट का प्रीव्यू करते समय दिखाई नहीं देंगे जब तक कि आप बिल्ड कमांड को फिर से नहीं चलाते।
+
+प्रीव्यू से बाहर निकलने के लिए (<kbd>Ctrl</kbd> + <kbd>C</kbd>) का उपयोग करें और एक अन्य टर्मिनल कमांड चलाएं, जैसे कि डेव सर्वर को पुनः आरंभ करके [डेवलपमेंट मोड में कार्य करना](#डेवलपमेंट-मोड-में-कार्य-करें) पर वापस जाएं, जो आपके संपादन के साथ अपडेट होता है और आपके कोड परिवर्तनों का लाइव पूर्वावलोकन दिखाता है।
+
+<ReadMore>[एस्ट्रो सीएलआई](/hi/reference/cli-reference/) और टर्मिनल कमांड के बारे में अधिक पढ़ें, जिनका उपयोग आप एस्ट्रो के साथ निर्माण करते समय करेंगे।</ReadMore>
+
+:::टिप
+आप बहुत ज़्यादा कोड जोड़ने या बदलने से पहले [अपनी नई साइट को तुरंत तैनात करना](/hi/guides/deploy/) चाह सकते हैं। यह आपकी साइट का न्यूनतम, कार्यशील संस्करण प्रकाशित करने में सहायक है और बाद में अपनी तैनाती के समस्या निवारण में आपका अतिरिक्त समय और प्रयास बचा सकता है।
+:::
```

Comment:
The syntax after `:::` shouldn't be translated because it specifies the type of aside to use, in this case Starlight will handle the translation for it, so it will show up properly in Hindi if someone already translated that!
```suggestion
:::tip
आप बहुत ज़्यादा कोड जोड़ने या बदलने से पहले [अपनी नई साइट को तुरंत तैनात करना](/hi/guides/deploy/) चाह सकते हैं। यह आपकी साइट का न्यूनतम, कार्यशील संस्करण प्रकाशित करने में सहायक है और बाद में अपनी तैनाती के समस्या निवारण में आपका अतिरिक्त समय और प्रयास बचा सकता है।
:::
```